code/game/machinery/computer/arcade_games/recruiter.dm

```diff
@@ -269,6 +298,18 @@
 /obj/machinery/computer/arcade/recruiter/attack_hand(mob/user)
 	ui_interact(user)
 
+/obj/machinery/computer/arcade/recruiter/emag_act(mob/user)
+	if(emagged)
+		return
+	if(user)
+		to_chat(user, "<span class='notice'>You override the menu and revert the game to its previous version.</span>")
+		add_hiddenprint(user)
+	game_status = RECRUITER_STATUS_START
+	name = "NT Recruiter Simulator HARDCORE EDITION"
+	desc = "The version utilized by the company to train real recruiters, comes with realistic consequences and double the applications."
```

Comment:
This one's up to you, I just felt like it could be fleshed out a bit more.
```suggestion
	desc = "The advanced version of Nanotrasen's recruiting simulator, used to train the highest echelon of Nanotrasen recruiters. Has double the application count, and supposedly includes some routines to weed out the less skilled."
```